srcpkgs/sigtool/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'sigtool'
+pkgname=sigtool
+version=0.6.2
+revision=1
+build_style=go
+go_import_path=github.com/opencoff/sigtool
+short_desc="Go version of OpenBSD's signify(1)"
+maintainer="Michael Aldridge <maldridge@VoidLinux.eu>"
```

Comment:
Because that address is buried somewhere in a config file I haven't edited yet to fix it.  As I'm not actually going to take this PR to a merge I'm not too concerned, but I'll dig through and figure out what still has that address set in it.